openbas-api/src/main/java/io/openbas/rest/inject/service/InjectService.java

```diff
@@ -92,23 +91,15 @@ public void deleteAll(List<Inject> injects) {
 
   public Map<Asset, Boolean> resolveAllAssetsToExecute(@NotNull final Inject inject) {
     Map<Asset, Boolean> assets = new HashMap<>();
-    inject
-        .getAssets()
-        .forEach(
-            (asset -> {
-              assets.put(asset, false);
-            }));
+    inject.getAssets().forEach((asset -> assets.put(asset, false)));
```

Comment:
Map<Asset, Boolean> assets = inject.getAssets().stream().collect(Collectors.toMap(Asset, false));

(forEach is often slower than streams)

Reply:
I didn't know that. Thx !